### PR TITLE
Release 1.2.78

### PR DESCRIPTION
**Pull request by mcmcphillips**
## MADiE PR

Jira Ticket: [MAT-8510](https://jira.cms.gov/browse/MAT-8510)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [x] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [x] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: react/components/RichTextEditor/index.jsx [hunk 1]
```diff
@@ -429,38 +429,53 @@ const RichTextEditor = ({
...
+            return ["del", HTMLAttributes, 0];
+          },
+        }),
+      ],
+      editorProps: {
+        attributes: {
+        tabIndex: '0',
+        role: 'textbox',
+        id,
+        'data-testid': 'rich-text-editor-content',
```
Comment (adongare) [inline react/components/RichTextEditor/index.jsx]: this doesn't have to be unique, right if there are multiple RTEs on page?

**Review by adongare — DISMISSED** [review]

**Review by sb-prateekkeerthi — DISMISSED** [review]

File: react/components/RichTextEditor/index.jsx (hunk 1 above)
Comment (mcmcphillips) [reply react/components/RichTextEditor/index.jsx]: It has to be unique, but I'm also afraid of how many tests this is going to break for QA by changing the data-testId. 

I'm going to modify it and just notify them

**Review by sb-cecilialiu — DISMISSED** [review]

**Review by adongare — DISMISSED** [review]

**Review by chubert-sb — APPROVED** [review]
